From 71651c31aad4499153b6e3dbb47cbb86bca15d94 Mon Sep 17 00:00:00 2001
From: Chun-wei Fan <fanchunwei@src.gnome.org>
Date: Mon, 8 Jun 2015 10:38:19 +0800
Subject: [PATCH 3/3] src/dispatch_common.h: Add PUBLIC Annotation Where Needed

...so that the build will not fail on Visual Studio builds as the function
definition for epoxy_conservative_gl_version() in src/dispatch_common.c is
already annotated with PUBLIC.  Again, Visual Studio requires that the
function prototype be annotated with dllexport/dllimport annotations when
the function definition is annotated, if the function prototype exists.
---
 src/dispatch_common.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/dispatch_common.h b/src/dispatch_common.h
index eea4e8f..3a4bf1f 100644
--- a/src/dispatch_common.h
+++ b/src/dispatch_common.h
@@ -158,7 +158,7 @@ void *epoxy_get_proc_address(const char *name);
 void *epoxy_get_core_proc_address(const char *name, int core_version);
 void *epoxy_get_bootstrap_proc_address(const char *name);
 
-int epoxy_conservative_gl_version(void);
+PUBLIC int epoxy_conservative_gl_version(void);
 bool epoxy_conservative_has_gl_extension(const char *name);
 int epoxy_conservative_glx_version(void);
 bool epoxy_conservative_has_glx_extension(const char *name);
-- 
2.3.5.windows.8

